app/javascript/util/languageUtil.tsx

```diff
@@ -113,3 +114,16 @@ export const getPathWithoutLanguagePrefix = (path: string): string => {
  */
 export const getCurrentLanguage = (path: string | undefined): LanguagePrefix =>
   getRoutePrefix(path || "") || LanguagePrefix.English
+
+/**
+ * Get a renderable version of a translated string with e.g. a link in it as an alternative to using <Markdown />
+ */
+export function renderWithInnerHTML(translatedString: string) {
```

Comment:
@emilyjablonski, curious to hear your thoughts on this when you're back. Another eng on our team was concerned about performance impacts of using <Markdown/> so I added this helper fn